claims.go

```diff
@@ -53,6 +53,7 @@ type Claims interface {
 	Claims() *ClaimsData
 	Encode(kp nkeys.KeyPair) (string, error)
 	ExpectedPrefixes() []nkeys.PrefixByte
+	Migrated() bool
```

Comment:
How is this used. I see it is now in all the claims, but i don't see where we use it. Do we need this?